### PR TITLE
fix(Table): withTableSorting can get sort value using column's ID written as path

### PR DESCRIPTION
**Pull request by chervyakovru**
Just duplicate get logic from main Table code: https://github.com/gravity-ui/uikit/blob/main/src/components/Table/Table.tsx#L171

### CODE REVIEW

File: src/components/Table/hoc/withTableSorting/withTableSorting.tsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ export function withTableSorting<I extends TableDataItem, E extends {} = {}>(
     const displayName = `withTableSorting(${componentName})`;
 
     function defaultCompareFunction(itemA: I, itemB: I, columnId: string) {
-        if (itemA[columnId] === itemB[columnId]) {
+        if (_get(itemA, columnId) === _get(itemB, columnId)) {
```
Comment (ogonkov) [inline src/components/Table/hoc/withTableSorting/withTableSorting.tsx:44]: What was the problem here?

Comment (chervyakovru) [reply src/components/Table/hoc/withTableSorting/withTableSorting.tsx:44]: for example: 
in the table we use Item with type: `{foo: {bar: string}, foo2: string, foo3: string}`
for columns Ids we use `['foo.bar', 'foo2', 'foo3']`

and right now, if we want sort table by first column, in the if we will get `undefined` from both items. Adding `_get` solve this issue

Comment (ogonkov) [reply src/components/Table/hoc/withTableSorting/withTableSorting.tsx:44]: Then it looks like new feature, not a fix

```
feat(Table): add support for sort by nested properties
```

**Comment by chervyakovru** [comment]
@ogonkov @krozhkov
Hi! Can we merge it?)

**Comment by ogonkov** [comment]
Please rebase PR

**Comment by Raubzeug** [comment]
Please check linter status, it fail now.

File: src/components/Table/hoc/withTableSorting/withTableSorting.tsx (hunk 1 above)
Comment (Raubzeug) [reply src/components/Table/hoc/withTableSorting/withTableSorting.tsx:47]: It seem it will be useful to add nested properties to stories and tests.

**Review by Raubzeug — CHANGES_REQUESTED** [review]

**Review by Raubzeug — APPROVED** [review]

**Review by Raubzeug — APPROVED** [review]

**Review by amje — APPROVED** [review]

**Pull request merged**